lib/video_info/providers/vkontakte.rb

```diff
@@ -8,15 +9,15 @@ class Vkontakte < Provider
       attr_accessor :video_owner
 
       def self.usable?(url)
-        url =~ /(vk\.com)|(vkontakte\.ru)/
+        !!(url =~ /(vk\.com)|(vkontakte\.ru)/)
       end
 
       def provider
-        'Vkontakte'
+        "Vkontakte"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.